Editor/BacktraceConfigurationEditor.cs

```diff
@@ -137,9 +137,16 @@ public override void OnInspectorGUI()
 
 
 #if UNITY_ANDROID || UNITY_IOS || UNITY_STANDALONE_WIN
+                        SerializedProperty captureNativeCrashes = serializedObject.FindProperty("CaptureNativeCrashes");
                         EditorGUILayout.PropertyField(
-                            serializedObject.FindProperty("CaptureNativeCrashes"),
+                            captureNativeCrashes,
                             new GUIContent(BacktraceConfigurationLabels.CAPTURE_NATIVE_CRASHES));
+#if !UNITY_2018_4_OR_NEWER
+                        if (captureNativeCrashes.boolValue)
+                        {
+                            EditorGUILayout.HelpBox("Native crsah reporter will be disabled for Untiy 2018 (and older) version that use NDK16b. Please contact Backtrace support to learn more about crash reporting tool.", MessageType.Warning);
```

Comment:
crsah -> crash

Reply:
Untiy -> Unity

Reply:
"Please contact Backtrace support to learn more about crash reporting tool" -> "Please contact Backtrace support for additional details."

Reply:
"#if !UNITY_2018_4_OR_NEWER" implies it will not be shown for 2018_4?

Reply:
version -> version

Reply:
I guess remove the parentheses around "and older" too